### PR TITLE
Add support for serverless_function_region and ignore_command on Project resources

### LINKED ISSUE #28: Feature Request - Git Integration Ignore Build Step
hello guys,
how about small and very useful feature for monorepos ??
in my case i want to add project in monorepo that has 25 frontends in Vercel and would be pain to manually add this ignore step

https://vercel.com/docs/concepts/projects/overview#ignored-build-step

### LINKED ISSUE #30: Feature Request - Node Version
Hi team, it would be useful to control the node version configuration in Terraform. I just upgraded my project to V16, and would love to have the config in code

### PR DESCRIPTION
**Pull request by dglsparsons**
Closes #28

### CODE REVIEW

File: vercel/resource_project_test.go [hunk 1]
```diff
@@ -47,6 +47,9 @@ func TestAcc_Project(t *testing.T) {
 							resource.TestCheckResourceAttr("vercel_project.test", "output_directory", ".output"),
 							resource.TestCheckResourceAttr("vercel_project.test", "public_source", "true"),
 							resource.TestCheckResourceAttr("vercel_project.test", "root_directory", "ui/src"),
+							resource.TestCheckResourceAttr("vercel_project.test", "ignore_command", "echo 'wat'"),
```
Comment (ecklf) [inline vercel/resource_project_test.go:50]: :wat:

Comment (dglsparsons) [reply vercel/resource_project_test.go:50]: :wat2:

**Review by ecklf — DISMISSED** [review]

File: client/project_get.go [hunk 2]
```diff
@@ -42,13 +42,14 @@ func (r *ProjectResponse) Repository() *Repository {
 
 // ProjectResponse defines the information vercel returns about a project.
```
Comment (ecklf) [inline client/project_get.go:43]:
```suggestion
// ProjectResponse defines the information Vercel returns about a project.
```

File: client/project_get.go [hunk 3]
```diff
@@ -61,10 +62,11 @@ type ProjectResponse struct {
...
-	PublicSource    *bool   `json:"publicSource"`
-	RootDirectory   *string `json:"rootDirectory"`
+	Name                     string  `json:"name"`
+	OutputDirectory          *string `json:"outputDirectory"`
+	PublicSource             *bool   `json:"publicSource"`
+	RootDirectory            *string `json:"rootDirectory"`
+	ServerlessFunctionRegion *string `json:"serverlessFunctionRegion"`
 }
 
 // GetProject retrieves information about an existing project from vercel.
```
Comment (ecklf) [inline client/project_get.go:72]:
```suggestion
// GetProject retrieves information about an existing project from Vercel.
```

File: client/project_update.go [hunk 4]
```diff
@@ -16,14 +16,16 @@ import (
...
+	Framework                   *string `json:"framework"`
+	InstallCommand              *string `json:"installCommand"`
+	Name                        *string `json:"name,omitempty"`
+	OutputDirectory             *string `json:"outputDirectory"`
+	PublicSource                *bool   `json:"publicSource"`
+	RootDirectory               *string `json:"rootDirectory"`
+	ServerlessFunctionRegion    *string `json:"serverlessFunctionRegion"`
 }
 
 // UpdateProject updates an existing projects configuration within vercel.
```
Comment (ecklf) [inline client/project_update.go:31]:
```suggestion
// UpdateProject updates an existing projects configuration within Vercel.
```

**Review by ecklf — APPROVED** [review]

**Pull request merged**